lib/mission_control/job_argument_filter.rb

```diff
@@ -0,0 +1,23 @@
+class MissionControl::JobArgumentFilter
```

Comment:
This is a class, but it could be a module because it has no instance methods 🤔  I think, alternatively, it could be a class that receives the configuration in the constructor, and then we would just have `MissionControl.job_arguments_filter = MissionControl::JobArgumentFilter.new(filter_arguments )` and use that like `MissionControl.job_arguments_filter.apply_to(args)` or something like that.